mne/preprocessing/ica.py

```diff
@@ -270,6 +271,22 @@ def __repr__(self):
 
         return '<ICA  |  %s>' % s
 
+    def _check_for_unsupported_ica_channels(self, picks, info):
```

Comment:
this doesn't use `self` at all, so it should be a small helper function instead of a method